src/movie/Home.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { MovieSearchBanner, MovieList, MovieDetails } from '.';
+
+export class Home extends Component {
+  constructor() {
+    super();
+    this.state = { query: '', movie: null };
+  }
+
+  handleQueryChange = (query) => {
+    query = query.toLowerCase();
+    this.setState({ query: query });
+  };
+
+  handleDetailsClick = (movie) => {
+    setTimeout(() => window.scrollTo({ left: 0, top: 0, behavior: 'smooth' }), 10);
+    this.setState({ movie: movie });
+  };
+
+  handleSearchIconClick = () => {
+    this.setState({ movie: null });
+  };
+
+  render() {
+    return (
+      <div className="router-page home-page">
+        {this.state.movie ? (
+          <MovieDetails
+            movie={this.state.movie}
+            onSearchIconClick={this.handleSearchIconClick}
+          />
+        ) : (
+          <MovieSearchBanner onQueryChange={this.handleQueryChange} />
+        )}
+        }
```

Comment:
this is not an extra curly brace by accident? please double-check it

Reply:
Yes it was. Will remove it.